src/main/java/net/syscon/elite/service/transformers/PropertyContainerTransformer.java

```diff
@@ -0,0 +1,34 @@
+package net.syscon.elite.service.transformers;
+
+import net.syscon.elite.api.model.CourtCase;
+import net.syscon.elite.api.model.CourtHearing;
+import net.syscon.elite.api.model.Location;
+import net.syscon.elite.api.model.PropertyContainer;
+import net.syscon.elite.repository.jpa.model.CaseStatus;
+import net.syscon.elite.repository.jpa.model.LegalCaseType;
+import net.syscon.elite.repository.jpa.model.OffenderCourtCase;
+import net.syscon.elite.repository.jpa.model.OffenderPropertyContainer;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Simple transform object to help reduce boiler plate mapping of entity {@link OffenderCourtCase} to model object {@link CourtCase}.
+ */
+public class PropertyContainerTransformer {
+
+    public static PropertyContainer transform(final OffenderPropertyContainer propertyContainer) {
+        return PropertyContainer.builder()
+                .location(Location.builder()
```

Comment:
Think there is already a transformer for locations ?

Reply:
There is, just in a different place. I've updated :)